integration-tests/integration/manageCourts.cy.js

```diff
@@ -5,6 +5,10 @@ const HomePage = require('../pages/homePage')
 context('A user can view the manage courts page', () => {
   beforeEach(() => {
     cy.task('reset')
+
+    // Added stub - where else would it get its court list from?
+    cy.task('stubCourts')
+
     cy.window().then(win => {
```

Comment:
Could not see where the courts data was stubbed? Added this to provide data required for the test.